save-demo/build.gradle.kts

```diff
@@ -22,13 +22,4 @@ val diktatVersion: String = libs.versions.diktat.get()
 dependencies {
     implementation(projects.saveCloudCommon)
     implementation(libs.save.common.jvm)
-
-    implementation(libs.ktlint.core)
-    implementation(libs.ktlint.rulesets.standard)
```

Comment:
Are these still needed in `libs.versions.toml`? Also, where do we store actual versions (of diktat-cli)?

Reply:
I guess we do not need to store ktlint version. Will also remove it.

For now actual version of diktat-cli and ktlint-cli should be stored in resources dir, but it will be changed to storage in next PR